test/common.py

```diff
@@ -57,37 +58,29 @@ def run_tests(argv=UNITTEST_ARGS):
 IS_PPC = platform.machine() == "ppc64le"
 
 
-def _check_module_exists(name):
-    r"""Returns if a top-level module with :attr:`name` exists *without**
-    importing it. This is generally safer than try-catch block around a
-    `import X`. It avoids third party libraries breaking assumptions of some of
-    our tests, e.g., setting multiprocessing start method when imported
-    (see librosa/#747, torchvision/#544).
+def _check_module_exists(name, min_version=None):
+    r"""Returns if a top-level module with :attr:`name` exists and satisfy a
+    minium version of :attr:`min_version` (if set). Ideally, this should not do
+    an actual import to avoid third party libraries breaking assumptions of some
+    of our tests, e.g., setting multiprocessing start method when imported
+    (see librosa/#747, torchvision/#544). However, we guard our multiprocessing
+    tests with `mp = mp.get_context(...)` so this shouldn't be too big a
```

Comment:
I'm not too sure about the consequences of this. Does `pkg_resources.get_distribution("numpy").version` work? (found this on stackoverflow)